### PR TITLE
Update inactivity query in 2015 Activity Campaign.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

**Comment by johngian** [comment]
Just a heads up, this has already been executed in dev so we need to make sure that this is only live on dev ATM.

File: remo/reports/migrations/0019_activity_campaign_2015.py [hunk 1]
```diff
@@ -14,15 +14,28 @@ def forwards(self, orm):
...
 
         for user in inactive_users:
             user.groups.clear()
             user.groups.add(alumni_group)
 
+            report = user.ng_reports.all().order_by('-report_date')[:1]
+            if report:
+                date_left_program = report[0].report_date
+            else:
+                date_left_program = datetime.datetime.now().date()
```
Comment (johngian) [inline remo/reports/migrations/0019_activity_campaign_2015.py]: This doesn't look like a good workaround when there is no last report (meaning no report). I suggest we add ``None`` instead.

**Comment by johngian** [comment]
Code looks OK. r+

**Pull request merged**